diktat-analysis.yml

```diff
@@ -429,7 +429,10 @@
 # Checks that using unnecessary, custom label
 - name: CUSTOM_LABEL
   enabled: true
-# Checks that property in constructor doesn't contains comment
+# Checks that outer lambda has explicit parameter name
```

Comment:
I think we also need to update other `diktat-analysis.yml` files, that we have, some of them used for smoke tests, some need for internal usage

Reply:
Yes, there is a bug after the update to new Kotlin, that caused this unit test (that checked that all diktat-analysis.yml were updated) to be ignored https://github.com/analysis-dev/diktat/issues/1222

I will investigate it one day...

Reply:
That issue is already fixed, and once @nulls pulls new changes from master (actually, Github is experiencing degraded performance right now, so the button `Update branch` is missing on this PR 😅), that test should get triggered.

Reply:
fixed and merged. Thanks to @Cheshiriks